pyccel/ast/core.py

```diff
@@ -2386,7 +2386,6 @@ def __init__(
                 name = name[0]
             else:
                 name = DottedName(*name)
-
```

Comment:
You should try and checkout meaningless changes before committing. It makes it easier to see which files have been modified